src/coreclr/debug/createdump/CMakeLists.txt

```diff
@@ -81,13 +81,26 @@ else()
     add_dependencies(createdump pal_redefines_file)
 endif(CLR_CMAKE_HOST_OSX)
 
+if(NOT CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
     target_link_libraries(createdump
         corguids
         dbgutil
         # share the PAL in the dac module
         mscordaccore
         dl
     )
+else(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
+    target_link_libraries(createdump
```

Comment:
I would suggest have single linked libraries, and declare these required as LOONGARCH64 as separate like this:
```cmake
target_link_libraries(createdump
    corguids
    dbgutil
    # share the PAL in the dac module
    mscordaccore
    dl
)
if(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
    target_link_libraries(createdump
        pthread
        rt
        lzma
    )
endif()
```

Reply:
Thanks for your review!

Reply:
I think whole changes in this file can be moved to https://github.com/dotnet/runtime/pull/62886 to make this PR sef-contained.